frontend/js/src/listens/ListenCard.tsx

```diff
@@ -106,20 +107,28 @@ export default class ListenCard extends React.Component<
 
     this.state = {
       isCurrentlyPlaying: false,
+      saveDataMode: false,
     };
   }
 
   async componentDidMount() {
     window.addEventListener("message", this.receiveBrainzPlayerMessage);
-    await this.getCoverArt();
+    const { userPreferences } = this.context;
+    if (userPreferences?.saveDate) {
+      this.setState({ saveDataMode: true });
```

Comment:
There should be no need to set this value in the state.
In `componentDidUpdate` below you can extract the property using `const { userPreferences } = this.context;` just like you do here.

Reply:
Fixed